eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -86,6 +86,7 @@ stages:
       excludeOmniSharpTests: true        # ✅
       runOnline: true                    # ✅
       useMonoRuntime: false              # 🚫
+      runTests: true                     # ✅
```

Comment:
Nit: The configuration parameters are alphabetized here, would you mind preserving the style?